lms/djangoapps/ccx/migrations/0004_change_ccx_coach_to_staff.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import logging
+
+from django.db import migrations
+# We're doing something awful here, but it's necessary for the greater good:
+from django.contrib.auth.models import User
```

Comment:
AFAIK, using the User model directly in the migration to swap roles isn't so bad, so I removed the comment for now.

CC @haikuginger: When you wrote the original comment here, what were you referring to?